backend/internal/types/occurrence.go

```diff
@@ -95,6 +96,12 @@ var SeqIdField = Field{
 	CanBeFiltered: true,
 	Table:         OccurrenceTable,
 }
+var TaskIdField = Field{
+	Name:          "task_id",
+	CanBeSelected: false,
```

Comment:
should be true since it's in the default fields

Reply:
Addressed in https://github.com/radiant-network/radiant-portal/pull/320/commits/b9bc588237a596ff60f1b5d35f38868730b75f8e